card/card.gemspec

```diff
@@ -34,8 +34,9 @@ Gem::Specification.new do |s|
 
     [ 'smartname',    '0.2.3'   ],
 
+    [ 'activerecord-session_store'],
     [ 'uuid',         '~> 2.3'  ],
-    [ 'paperclip',    '~> 2.8'  ],
+    [ 'paperclip',    '~> 4.2'  ],
```

Comment:
Ah, didn't realize this is downstream of the Rails 4 update.  I guess this was missed in that branch, or more likely it just isn't merge yet.